src/pages/[platform]/reference/amplify_outputs/index.mdx

```diff
@@ -0,0 +1,137 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'About amplify_outputs.json',
+  description: 'Reference for amplify_outputs.json',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ]
+};
+
+export async function getStaticPaths() {
+  return getCustomStaticPath(meta.platforms);
+}
+
+export async function getStaticProps() {
+  async function fetchBackendOutputsSchema() {
+    const request = new Request("https://raw.githubusercontent.com/aws-amplify/amplify-backend/main/packages/client-config/src/client-config-schema/schema_v1.json")
```

Comment:
Not super comfortable with this, can we inline it next to this file for now, and follow up with making it dynamic?